test/spoom/cli/bump_test.rb

```diff
@@ -367,6 +367,29 @@ class A; end
         assert_equal("true", Sorbet::Sigils.file_strictness("#{@project.path}/file1.rb"))
         assert_equal("false", Sorbet::Sigils.file_strictness("#{@project.path}/vendor/file2.rb"))
       end
+
+      def test_bump_count_errors
+        @project.write("file1.rb", <<~RB)
+          # typed: false
+          class Foo
+            def foo
+            end
+          end
+
+          Foo.new.foos
+        RB
+
+        out, err, status = @project.bundle_exec("spoom bump --no-color --count-errors")
+        assert_empty(err)
+        assert_equal(<<~OUT, out)
+          Checking files...
+
+          Found 1 type checking error(s).
```

Comment:
```suggestion
          Found 1 type checking error(s)
```

We never use `.` at the end of the message in `spoom` (you can open another PR if you want to change them all 😂)

Also I think there is already some logic round the `s` plural form for `X file(s?) bumped`, let's do the same for `error(s?)` so the messages are uniform please 🙏

Reply:
Done!